upup/pkg/fi/cloudup/awstasks/load_balancer.go

```diff
@@ -529,6 +536,24 @@ func (_ *LoadBalancer) RenderAWS(t *awsup.AWSAPITarget, a, e, changes *LoadBalan
 			return fmt.Errorf("Unable to find newly created ELB %q", loadBalancerName)
 		}
 		e.HostedZoneId = lb.CanonicalHostedZoneNameID
+
+		// Set ELB listener's Security Policy
+		if l, ok := e.Listeners["443"]; ok && len(l.PolicyNames) != 0 {
```

Comment:
Rather than looking for the listener on port 443 perhaps it would be more flexible to perform this logic on any listener that has a Protocol of `SSL`?

Reply:
We only apply security policies when specifying an ACM certificate. However, we set the listener with 443 port when the ACM certificate is existent. So I only set security policies for the listener with 443 port here.

https://github.com/kubernetes/kops/blob/d90c90cfa41bc8b161dc5475c3280fb0f133c12e/pkg/model/awsmodel/api_loadbalancer.go#L112-L114